Analysis/HiggsTauTau/src/HTTRun2AnalysisTools.cc

```diff
@@ -1274,7 +1174,7 @@ push_back(sample_names_,this->ResolveSamplesAlias("data_samples"));
       std::string qcd_extrap_sel = "!os && " + this->ResolveAlias("sel");
       //Default QCD method for fully hadronic is different...
       if(ch_ == channel::tt && method == 8){
-        qcd_sdb_sel =  do_ss_ ? "!os &&" + this->ResolveAlias("sel") : "os && " + this->ResolveAlias("sel");
+        qcd_sdb_sel = "os && " + this->ResolveAlias("sel");
```

Comment:
again I don't know why this was changed but i've changed this to match the master